src/DssDirectDepositHub.sol

```diff
@@ -0,0 +1,460 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021-2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.12;
+
+interface D3MPoolLike {
+    function deposit(uint256) external;
+    function withdraw(uint256) external;
+    function transfer(address, uint256) external returns (bool);
+    function transferAll(address) external returns (bool);
+    function accrueIfNeeded() external;
+    function assetBalance() external returns (uint256);
+    function maxDeposit() external view returns (uint256);
+    function maxWithdraw() external view returns (uint256);
+    function active() external view returns (bool);
+}
+
+interface D3MPlanLike {
```

Comment:
Since this is not an external dependency we should import here.

`import { ID3MPlan } from "../plans/ID3MPlan.sol";`

Reply:
changed in https://github.com/makerdao/dss-direct-deposit/commit/5945130d2783313fe477ab9390ff8d5b16a8bc29